tests_bucklescript/__tests__/pokedexScalars.re

```diff
@@ -0,0 +1,35 @@
+module MyQuery = [%graphql
+  {|
+    query pokemon($id: String, $name: String) {
+      pokemon(name: $name, id: $id) {
+        id
+        name
+      }
+    }
+  |};
+  {schema: "../pokedex_schema.json"}
+];
+
+Jest.(
+  describe("Scalars as arguments", () => {
+    open Expect;
+    open! Expect.Operators;
+
+    test("Allows you to omit nullable arguments", () =>
+      expect(MyQuery.make(~name="Pikachu", ())##variables)
+      == Js.Json.parseExn({| {
```

Comment:
Please change to |> based tests. I think it's more readable that way.

Reply:
👍